elements/power.lua

```diff
@@ -160,33 +164,30 @@ local Update = function(self, event, unit)
 	end
 
 	power.disconnected = disconnected
-
+	power.tapped = tapped
+	
 	local ptype, ptoken, altR, altG, altB = UnitPowerType(unit)
-	local t = self.colors.power[ptoken]
+	local t = self.colors.power[ptoken or ptype or 'MANA']
 	local r, g, b
 	local atlas = power.atlas or t.atlas
-	local bg = power.bg
-
+	
 	if(power.useAtlas and atlas and displayType ~= ALTERNATE_POWER_INDEX) then
 		r, g, b = t[1], t[2], t[3]
```

Comment:
Or change line 217 from:
`if(t) then`
to
`if(t and nor r) then`
This should cover Sticklord's concern, which is valid, I missed that one.